public/blacklist.json

```diff
@@ -0,0 +1,58 @@
+[
+  "0x64aa3364f17a4d01c6f1751fd97c2bd3d7e7f1d5",
```

Comment:
Let's either make this a TS file that exports the array with a comment linking to the source of this array, or document that link in the PR description